ikarus/finiteelements/mechanics/kirchhoffloveshell.hh

```diff
@@ -196,17 +201,12 @@ namespace Ikarus {
 
     std::shared_ptr<const Geometry> geo_;
```

Comment:
Is there a `private:` missing here? Maybe also in the old code sorry :/

Reply:
Ah, it is missing for all the elements. I will add it.